docs/src/templates/docs.js

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+import { graphql } from "gatsby"
+import { MDXRenderer } from "gatsby-plugin-mdx"
+import SEO from "../components/seo"
+
+const Docs = ({ data }) => {
```

Comment:
When we paired yesterday, this template received `element` directly from `gatsby-plugin-mdx` and we rendered the sidebar layout directly. It's not just passing `frontmatter` into `SEO` and rendering `body`.